### ISSUE TITLE
"Package ac-ispell is unavailable. Is the package name misspelled?"

### ISSUE THREAD

**Issue by giuscri** [syl20bnr/spacemacs#11096]
#### Description :octocat:
Spacemacs prints an error message when starting.

#### Reproduction guide :beetle:
- Start Emacs
- Wait for _start-up procedure_ to complete

*Observed behaviour:* :eyes: :broken_heart:
In `*spacemacs*` buffer, this error message is printed
```
...
--> installing package: ac-ispell@auto-completion... [1/1]
Package ac-ispell is unavailable. Is the package name misspelled?
```

*Expected behaviour:* :heart: :smile:
The above message is _not_ printed.

#### System Info :computer:
- OS: gnu/linux
- Emacs: 26.1
- Spacemacs: 0.300.0
- Spacemacs branch: develop (rev. f2a4cc87c)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(graphviz csv yaml markdown javascript python helm
          (auto-completion :variables auto-completion-complete-with-key-sequence-delay 0)
          emacs-lisp git neotree
          (shell :variables shell-default-height 30 shell-default-position 'bottom)
          themes-megapack)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG IMAGEMAGICK SOUND GPM DBUS GSETTINGS NOTIFY ACL GNUTLS LIBXML2 FREETYPE M17N_FLT LIBOTF XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 MODULES THREADS LIBSYSTEMD LCMS2

**Comment by Miciah** [comment]
Looks like a problem with MELPA.  https://melpa.org/#/ac-ispell says, "Package not found", but https://github.com/melpa/melpa/blob/master/recipes/ac-ispell has not changed since 2013.

**Comment by Miciah** [comment]
This is a duplicate of issue #11095.

**Comment by Miciah** [comment]
Upstream issue: https://github.com/melpa/melpa/issues/5657

**Comment by sdwolfz** [comment]
I've disabled the package for now. Will revert my changes once it's restored.

**Comment by heegoo-han** [comment]
Hmm, I've also encountered this problem. Gonna disable the package for now

**Comment by crazymousethief** [comment]
@sdwolfz It seems that `ac-ispell` is once again available.

**Comment by giuscri** [comment]
fixed, as of 6a724e416de4c3529f30edb45213a793a13cb959.
